executor/join.go

```diff
@@ -747,9 +785,21 @@ func (e *HashJoinExec) buildHashTableForList(buildSideResultCh <-chan *chunk.Chu
 			return err
 		}
 	}
+
+	if e.bloomFilter != nil {
+		e.bloomFilter.Build()
+	}
+
 	return nil
 }
 
+// PutChunkToBloom puts hash values in hctx to bloom filter.
+func (e *HashJoinExec) PutChunkToBloom(hctx *hashContext) {
+	for _, hash := range hctx.hashVals {
+		e.bloomFilter.LazyInsertU64(hash.Sum64())
```

Comment:
How much faster is `LazyInsertU64` than `InsertU64` here?

Reply:
The function is used to solve that build side rowNum estimate inaccurately, so I save all the hash val and build bloom filter in the end.